app-accessibility/orca/orca-43.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
I'd like to add Python 3.11 here too, but I see that we're blocked by some of the accessibility dependencies. Let me ping the maintainer like I have to do every orca bump...